addons/web_editor/static/src/js/common/ace.js

```diff
@@ -201,8 +201,8 @@ var ViewEditor = Widget.extend({
             defaultBundlesRestriction: [],
         });
 
-        this.resources = {xml: {}, scss: {}, js: {}};
-        this.editingSessions = {xml: {}, scss: {}, js: {}};
+        this.resources = { xml: {}, scss: {}, js: {} };
+        this.editingSessions = { xml: {}, scss: {}, js: {} };
```

Comment:
Don't add such noise in this commit. One of two fixing liting is ok but I see many:
- It should be in its own commit
- Here I am not even sure we enforce that convention? Or is the Odoo linter configuration doing that?

Reply:
> Or is the Odoo linter configuration doing that?

Is it @sleepy-monax ? I was going to suggest something in the Odoo task and saw this one, was interested by the answer as I am still using my own linter config.